### ISSUE TITLE
Where to meet

### ISSUE THREAD

**Issue by Gersh305** [ecn310/course-project-zipcentercrime#20]
If we cannot meet in person here is where the zoom link or secondary meeting location will be sent

**Comment by Wes-Mac** [comment]
@kbuzard I was wondering if you would be able to see if the people in charge of key card access to extend my access to room 040 until the 17th?

**Comment by kbuzard** [comment]
> @kbuzard I was wondering if you would be able to see if the people in charge of key card access to extend my access to room 040 until the 17th?

Sure! But I'd like to only bother them once, so please check with the whole team and see if anyone else needs access extended. I'll send an email first thing tomorrow morning to ITS.

**Comment by sortizh** [comment]
@rpseely Could you meet our group tomorrow (12/13) at 3:00 pm in the normal lab room? To go over our reproducibility package and the paper's contents/formatting. I can send you our paper tomorrow morning for you to overview before we meet up!

**Comment by rpseely** [comment]
@sortizh Yes, sounds good. I am going to put this in the big discussion so that others can see that this time slot is filled for your team. I will have to leave by 4:50 to meet with the taxes-tariffs team.

**Comment by rpseely** [comment]
@sortizh Can you send me what you have for your paper so I can take a look before 3?

**Comment by sortizh** [comment]
@rpseely what is the best way for me to send it over? As a pdf, zip file, or copy and paste?

**Comment by sortizh** [comment]
@rpseely I'll send you the pdf for now! [Here it is](https://github.com/ecn310/course-project-zipcentercrime/blob/main/course_project_zipcentercrime_ryan.pdf)

**Comment by rpseely** [comment]
@sortizh Thank you! Is there anything in particular you want me to look at?

**Comment by sortizh** [comment]
@rpseely For the results section and conclusion, the content. For the introduction and discussion, prose (specifically looking for passive language, and grammar to improve on) or sections that are missing some of the logical content. I think in person I would like to consult about what our data appendix should look like.
